java-file-formats/word-library/Working-with-LaTeX.md

```diff
@@ -0,0 +1,1332 @@
+---
+title: Working with LaTeX | DocIO | Syncfusion
+description: Learn how to create mathematical equations in a Word document using LaTeX with the Java Word library without Microsoft Word.
+platform: java-file-formats
+control: Word Library
+documentation: UG
+---
+
+# Create Equation using LaTeX
+The Java Word library allows to create mathematical equation in Word document using **LaTeX**.
+
+## Accent
+
+Add **accent** equation to a Word document using the LaTeX through appendMath API.
+
```

Comment:
Highlight the API name as like this(\`) `appendMath`

Reply:
Highlighted the API name as like appendMath() in places